scanners/helpers.js

```diff
@@ -0,0 +1,284 @@
+// SPDX-FileCopyrightText: the secureCodeBox authors
```

Comment:
Lot of going on here... Is this file copied from somewhere else or written from scratch? In the latter case we should also have at least some basic unit tests for the more complex functions (with for-loops and such) to prevent simple errors.

Reply:
It's copied over as mentioned in the PR description:
> As a side effect of the transition to the use of makefile. We've had to have a two copies of helpers.js. One in scanners/ folder and the other in tests/integration. The one in tests/integration can be deleted after the migration of the hooks integration tests is complete. (This is why the codeclimate check is unhappy)

But yes unit tests should be implemented.

Reply:
ok